### PR TITLE
feat - added product page

### PR DESCRIPTION
**Pull request by Balakumar3018**
yet to add responsive
preview of product page - https://deploy-preview-4--reverent-hypatia-c5af08.netlify.app/pages/products/product.html

screenshot-Screenshot 2022-02-22 at 11 33 59 AM

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -8,7 +8,6 @@ body{
 .navigation-bar{
     /* border: none; */
```
Comment (Aditi0505) [inline main.css]: Can remove commented code if not needed.

File: pages/products/product.css [hunk 2]
```diff
@@ -53,15 +50,16 @@
...
+    margin-top: -670px;
     width: 70%;
     padding: 16px;
-    /* display: flex; */
-    /* justify-content: space-evenly; */
-    /* flex-wrap: wrap; */
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
     float: right;
```
Comment (Aditi0505) [inline pages/products/product.css]: Any specific reason to use floats here? Could have used margin-right : auto maybe. Let me now your thoughts about this?

File: pages/products/product.html [hunk 3]
```diff
@@ -93,8 +94,166 @@ <h3>Sort by</h3>
...
+                          <button class="btn btn-primary">Add to Cart</button>
+                          <button class="btn btn-secondary">Order now</button>
+                        </div>
+                     </div>
+                    </div>
+                </div>
+                <div class="normal-card">
+                    <div class="badge-card basic-card">
+                        <div class="card-image-button">
+                          <img class="card-image" src="https://i.pravatar.cc/150?img=11" alt="card-image" width="250">
```
Comment (Aditi0505) [inline pages/products/product.html:120]: Can come up with a better alt description. It is a given that it is an image. You can make it more meaningful by adding the product name to it. Eg. card-product-sunglasses.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. rest looks good to me.
Also, please provide description and screenshots for better visualization.

File: pages/products/product.html (hunk 3 above)
Comment (Balakumar3018) [reply pages/products/product.html:120]: sure will add that

File: pages/products/product.css (hunk 2 above)
Comment (Balakumar3018) [reply pages/products/product.css]: not a specific reason. just wanted product cards to right of the sidebar. now i changed to margin-left:auto it s giving same result thanks

**Review by kirti2602 — APPROVED** [review]
Looks good to me. Can merge

File: main.css [hunk 4]
```diff
@@ -6,9 +6,7 @@ body{
     margin-top: -10px;
 }
 .navigation-bar{
-    /* border: none; */
```
Comment (bhavyaj12) [inline main.css:9]: Try using responsive lengths like rems.

File: main.css [hunk 5]
```diff
@@ -6,9 +6,7 @@ body{
...
     height: 60px;
-    /* width: 100%; */
 }
 .btn-login{
     height: 30px;
```
Comment (bhavyaj12) [inline main.css:12]: Here also

File: pages/products/product.css [hunk 6]
```diff
@@ -53,15 +50,16 @@
...
-    float: right;
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    margin-left: auto;
+    overflow-y: scroll;
+    height: 600px;
+}
+.normal-card{
+    padding: 8px;;
```
Comment (bhavyaj12) [inline pages/products/product.css]: Extra semi colon

File: pages/wishlist/wishlist.html [hunk 7]
```diff
@@ -15,26 +15,25 @@
...
               </span>
               <span class="avatar cart-icon">
-                  <a href="/pages/cart/cart.html"><i class="fa-solid fa-cart-shopping fa-lg"></i></a>
+                <a href="/pages/cart/cart.html"><i class="fa-solid fa-cart-shopping fa-lg"></i></a>
                   <span class="cart-badge "></span>
               </span>
             </div>
-        </div>
-        <h2 class="wishlist-text">Cart lists displayed here </h2>
+        </div>        <h2 class="wishlist-text">Cart lists displayed here </h2>
```
Comment (bhavyaj12) [inline pages/wishlist/wishlist.html]: Remove extra space before h2 tag

**Review by bhavyaj12 — COMMENTED** [review]
Added a few comments. Rest is good!

**Pull request merged**